docs/content/latest/contribute/core-database/clion-setup.md

```diff
@@ -23,19 +23,19 @@ There are two options for build systems that you can use with YugabyteDB, `cmake
 ### Configure a CLion project for YugabyteDB
 
 1. Click **File > Open…** to open the project root directory.
```

Comment:
If we're redoing this file anyway, let's do it as a proper markdown list: each entry starts with `1. `, put a blank line between steps, and indent any sub-paragraphs (like line 37) by four spaces (and a blank line before and after).

Reply:
Made these changes.